File: week1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int level;
    do 
    {
        level = get_int("Enter the number of levels:\n");
    }
    while (level < 1 || level > 8);
    
    
    for (int i = 0; i < level; i++)
    {

        for (int j = 0; j < level; j++)
        {
            
            if (level - i - 1 > j)
            {
                
                printf(" ");
                
            }
            else
            {
                    
                printf("#");
            }
        }
        printf("\n");
    }
    
    
}

File: week6/mario/more/mario.py
from cs50 import get_int


def main():
    x = get_right_int()


def get_right_int():
    while True:
        number = get_int("Print the number from 1 to 8: ")
        if number > 8 or number < 1:
            print("number is wrong")
        else:
            break

    for i in range(number):
        for j in range(number):
            if number - i - 1 > j:
                print(" ", end="")
            else:
                print("#", end="")
        print("  ", end="")
        for j in range(number):
            if j <= i:
                print("#", end="")
        print()


main()
